Update  5.1 ( Skipped $banID ) (#252)
NilPointer-Software/bdfd-wiki: src/bdscript/banID.md

# $banID
Bans a user using their ID.

## Syntax
```

> $ban[(reason;user ID)]

Reviewer: ```suggestion
$banID[reason;(user ID)]
```

```

## Parameters

> - `reason`: The reason for this action, which will be sent to the audit-log. _(Optional)_

Reviewer: ```suggestion
- `reason`: The reason for this action, which will be saved in the audit log. Can be left empty.
```

> The reason for the ban can be viewed using `$getBanResaon[]`.

> - `user ID`: Gets the user to ban. _(Optional)_

Reviewer: ```suggestion
- `user ID`: The user to ban. _(Optional)_ Can be omitted, then the ID will be taken from the last part of the command message.
```
